.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: GitHub Actions for Quantum Inspire Examples CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
Would be nice to disable this for a bit and try it out.

Reply:
As said, I tested locally with act (https://github.com/nektos/act/blob/master/README.md)